scripts/build_layers.sh

```diff
@@ -10,7 +10,14 @@ set -e
 
 LAYER_DIR=".layers"
 LAYER_FILES_PREFIX="datadog_lambda_node"
-NODE_VERSIONS=("10.15" "12.13" "14.15")
+
+if [ -z "$NODE_VERSION" ]; then
+    echo "Node version not specified, running for all node versions."
+    NODE_VERSIONS=("10.15" "12.13" "14.15")
```

Comment:
Can we just use the major version, i.e. `NODE_VERSION=10`? I think that would be easier to remember than `NODE_VERSION=10.15` and won't change as often.

Reply:
actually those versions are useful to get the right alpine tag to build the docker image 
see : ```docker build -t datadog-lambda-layer-node:$1 . --no-cache --build-arg image=node:$1-alpine```
So I think we can't get rid of them ?

Reply:
My concern is that there are some cases where we want to manually run this script. And it's going to be annoying to have to look up what the minor version number is every time! 😂 

Maybe in the "invalid version" warning, we could have the error message tell you the possible correct version options.

Another option would be to check the argument and convert to the correct alpine tag:

10 -> 10.5
12 -> 12.13
14 -> 14.15
other -> invalid

Reply:
I like the second option better but the first option would be OK too